README.md

```diff
@@ -59,6 +59,7 @@ The following commands should do everything for you:
 wget -O main.zip https://github.com/henne49/dbus-opendtu/archive/refs/heads/main.zip
 unzip main.zip "dbus-opendtu-main/*" -d /data
 mv /data/dbus-opendtu-main /data/dbus-opendtu
+rm /data/dbus-opendtu/current.log*
```

Comment:
I would recommend to do this automatically in the install and uninstall script.

For example:
```
# delete old logs if they exist
if [ -f /data/dbus-opendtu/current.log ]; then
    rm /data/dbus-opendtu/current.log*
fi
```

Reply:
Either check for all files ("*") and then also remove all. Or both not.
Do we have rotated log files (is "\*" needed) ?

Reply:
Answer to myself: Yes we have them. And it's ok like proposed: The won't be a current.log.1 if there is no current.log

Reply:
very good idea and put into install and uninstall